homeassistant/components/mqtt/strings.json

```
@@ -985,6 +991,27 @@
                 "pattern": "A valid regex pattern"
               },
               "name": "Advanced text entity settings"
+            },
+            "valve_payload_settings": {
+              "data": {
+                "payload_close": "Payload \"close\"",
+                "payload_open": "Payload \"open\"",
+                "payload_stop": "Payload \"stop\"",
+                "state_closed": "State \"closed\"",
+                "state_closing": "State \"closing\"",
+                "state_open": "State \"open\"",
+                "state_opening": "State \"opening\""
+              },
+              "data_description": {
+                "payload_close": "The payload sent when a \"close\" command is issued.",
+                "payload_open": "The payload sent when an \"open\" command is issued.",
+                "payload_stop": "The payload sent when an \"stop\" command is issued. Set this payload only if your lock supports the \"stop\" action.",
```

```suggestion
                "payload_stop": "The payload sent when a \"stop\" command is issued. Set this payload only if your lock supports the \"stop\" action.",
```

Just to make sure : "lock" in this string is correct? – Because everything above is about valves.